.github/advanced-issue-labeler.yml

```diff
@@ -0,0 +1,76 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
+# and should only be modified in that repository.
+
+# syntax - https://github.com/redhat-plumbers-in-action/advanced-issue-labeler#policy
+# Below keys map from the option used in issue form dropdowns to issue labels
+# Limitation:
+# Currently it's not possible to use strings containing ,␣ in the dropdown menus in the issue forms. 
+
+---
+
+policy:
+  - template: [bug.yml, feature.yml]
+    section:
+      - id: [frequency]
+        label:
```

Comment:
Do we also want to know how many users is affecting?, (all, many, just a few or once). I think is a different information than frequency and also can help us prioritise the issue.

Reply:
Yeah if we think that would be useful. Perhaps we can add that as a separate PR?
When i'm thinking about "reach", I'm mostly thinking about if it affect many apps versus many users of a single app. But it's surely relevant.